pages/changelog/v0.7.3-rc.mdx

```diff
@@ -0,0 +1,47 @@
+---
+date: 2024/05/14
+title: 🚀 LibreChat v0.7.3-rc
+description: The v0.7.3-rc release of LibreChat
+ogImage: /images/changelog/v0.7.2.jpg
```

Comment:
I'd suggest we remove this for the release candidates and keep the thumbnail for major releases only